### PR TITLE
Building completed coordinates

### LINKED ISSUE #298: Show hutblock coordinates when announcing building completion
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [X] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version: minecolonies-0.13.602-ALPHA-universal.jar
- Related Mods and their Versions:

## Is your feature request related to a problem?
My colony is quite large so when certain buildings are completed (such as higher level housing) I have to spend a long time trying to figure out where the upgrade occurred.

## Enhancement or Change description
When a builder has completed a building the X/Y/Z coordinates should appear in the message that is sent out.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

### PR DESCRIPTION
**Pull request by RyTechro**
Closes https://github.com/ldtteam/minecolonies-features/issues/298

# Changes proposed in this pull request:
- Added x, y and z when a builder completes a build

Schermafbeelding 2021-03-09 om 12 54 07

Alternatively it can report it as
Schermafbeelding 2021-03-09 om 13 26 14

 
Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -354,11 +354,11 @@
   "entity.worker.inventoryfullchestfull": "Please make space in my chests/racks. I am not able to dump my inventory.",
 
   "entity.builder.messagebuildstart": ": I've started work on %s!",
-  "entity.builder.messagebuildcomplete": ": I've finished building %s!",
-  "entity.builder.messagebuildcomplete_manual": ": I've finished building %s. You can assign me another task now!",
+  "entity.builder.messagebuildcomplete": ": I've finished building %s at x=%s y=%s z=%s",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "entity.builder.messagebuildcomplete": ": I've finished building the %s at %s %s %s!",
```
Not sure if we need the `xyz` there

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -354,11 +354,11 @@
...
+  "entity.builder.messagebuildcomplete_manual": ": I've finished building %s at x=%s y=%s z=%s. You can assign me another task now!",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "entity.builder.messagebuildcomplete_manual": ": I've finished building the %s at %s %s %s. You can assign me another task now!",
```

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 3]
```diff
@@ -354,11 +354,11 @@
...
   "entity.builder.messageremovalcomplete": ": I've finished deconstructing building %s! §6 To move the building while retaining its current level, use the 'Pick Up' button on the building's 'Build Options' GUI.",
-  "entity.builder.messagedecocomplete": "Builder %s: I've finished building %s!",
-  "entity.builder.messagedecocomplete_manual": "Builder %s: I've finished building %s. You can assign me another task now!",
+  "entity.builder.messagedecocomplete": "Builder %s: I've finished building %s at x=%s y=%s z=%s",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "entity.builder.messagedecocomplete": "Builder %s: I've finished building the %s at %s %s %s!",
```

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 4]
```diff
@@ -354,11 +354,11 @@
...
+  "entity.builder.messagedecocomplete_manual": "Builder %s: I've finished building %s at x=%s y=%s z=%s. You can assign me another task now!",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "entity.builder.messagedecocomplete_manual": "Builder %s: I've finished building the %s at %s %s %s. You can assign me another task now!",
```

**Review by Mekle001 — DISMISSED** [review]

**Pull request merged**